src/cheri/rvy64-encoding.adoc

```diff
@@ -0,0 +1,593 @@
+==== Bounds (EF, T, TE, B, BE) Encoding
+
+===== Concept
+
+ifdef::cheri_v9_annotations[]
+NOTE: *CHERI v9 Note:* The bounds mantissa width is different in {rvy32_uni_base_name}.
+Also, the old IE bit is renamed to Exponent Format (EF); the function of IE
+is the inverse of EF, i.e., IE=0 has the same effect as EF=1.
+
+NOTE: *CHERI v9 Note:* The mantissa width for RV32 was increased to 10.
+
+NOTE: *CHERI v9 Note:* The sense of the exponent is reversed, so an encoded
+value of 0 represents CAP_MAX_E, and CAP_MAX_E represents 0 from the previous
+specification.
+endif::[]
+
+The bounds encode the base and top addresses that constrain memory accesses.
+The capability can be used to access any memory location A in the range base
+&#8804; A < top. The bounds are encoded in a compressed format, so it is not
+possible to encode any arbitrary combination of base and top addresses. An
+invalid capability with {ctag} cleared is produced when attempting to construct a
+capability that is not _representable_ because its bounds cannot be correctly
+encoded. The bounds are decoded as described in
+xref:section_cap_encoding[xrefstyle=short].
+
+The bounds field has the following components:
+
+* *T:* Value substituted into the capability's address to decode the top
+address
+* *B:* Value substituted into the capability's address to decode the base
+address
+* *E:* Exponent that determines the position at which B and T are substituted
+into the capability's address
+* *EF:* Exponent format flag indicating the encoding for T, B and E
+    ** The exponent is stored in T and B if EF=0, so it is 'internal'
+    ** The exponent is zero if EF=1
+
+The bit width of T and B are defined in terms of the mantissa width (MW) which
+is set depending on capability encoding as shown in
+xref:mantissa_bit_width[xrefstyle=short].
+
+.Mantissa width (MW) values depending on capability encoding
+[#mantissa_bit_width,options=header,align="center",width="55%"]
+|==============================================================================
+^| Encoding   ^| MW
+^| {rvy32_uni_base_name} ^| {cap_rv32_mw_width}
+^| {rvy64_uni_base_name} ^| {cap_rv64_mw_width}
+|==============================================================================
+
+The exponent E indicates the position of T and B within the capability's
+address as described in xref:section_cap_encoding[xrefstyle=short]. The bit
+width of the exponent (EW) is set depending on the encoding. The
+maximum value of the exponent is calculated as follows:
+
+[source]
+----
+CAP_MAX_E = XLEN - MW + 2
+----
+
+The possible values for EW and CAP_MAX_E are shown in
+xref:exp_bit_width[xrefstyle=short].
+
+.Exponent widths and CAP_MAX_E depending on capability encoding
+[#exp_bit_width,options=header,align="center",width="55%"]
+|==============================================================================
+^| XLEN                 ^| EW ^| CAP_MAX_E
+^| {rvy32_uni_base_name} ^| {cap_rv32_exp_width}  ^| {cap_rv32_exp_max}
```

Comment:
I think this should move into the RV32Y encoding chapter
Just keep this section about RV64Y

Reply:
I'm fixing this